apps/root/src/common/components/news-banners/earn-banner/index.tsx

```diff
@@ -44,7 +44,7 @@ const EarnBanner = () => {
 
   return (
     <>
-      {hasEarnAccess ? (
+      {isEarnEnabled ? (
```

Comment:
Small bug here, when enabling earn env var, the earn-banner would still be showing the suscription banner for `earnEnabled: false` users